tests/tape/test_qnode.py

```diff
@@ -456,8 +471,8 @@ def circuit(p1, p2=y, **kwargs):
 
         result = draw(circuit)(p1=x, p3=z)
         expected = """\
- 0: ──RX(0.1)───RX(0.4)──╭C──╭┤ ⟨Z ⊗ X⟩ 
- 1: ──RY(0.06)───────────╰X──╰┤ ⟨Z ⊗ X⟩ 
+ 0: ──RX(0.1)───RX(0.4)──╭C──╭┤ ⟨Z ⊗ X⟩
+ 1: ──RY(0.06)───────────╰X──╰┤ ⟨Z ⊗ X⟩

```

Comment:
Hi @paniash, could you revert these changes here that are unrelated to the test you added above? Thanks!

Reply:
Apologies for such formatting changes. By default, my editor removes any trailing whitespaces.